src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
 // export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+    const { name, age, isMarried, partnerName, sex } = person;
+  
+    return (
+      <section className="Person">
+        <h2 className="Person__name">My name is {name}</h2>
+        {age !== undefined && <p className="Person__age">I am {age}</p>}
+        {isMarried ? (
+          <p className="Person__partner">
+            {partnerName} is my {sex === 'm' ? 'wife' : 'husband'}
+          </p>
+        ) : (
+          <p className="Person__partner">I am not married</p>
+        )}
```

Comment:
[CODE STYLE] - Readability is everything. You have a ternary operator here with quite a bit of code in each operand. It would be better to move each operand to a separate line. Also, consider creating a separate variable for the condition to improve readability.